common/app/views/fragments/containers/facia_cards/container.scala.html

```diff
@@ -79,6 +81,29 @@
                         </div>
                     }
                 }
+
+                @defining((
+                    containerDefinition.index == 0,
+                    frontId.getOrElse(pageId)
+                    )) { case (isFirstContainer, id) =>
+
+                    @if(mvt.ABNewHeaderVariant.isParticipating && isFirstContainer) {
+
+                        <ul class="tertiary-navigation mobile-only">
+
+                            @NewNavigation.TertiaryNav.getTertiaryNavLinks(id).map { footballNavItem =>
```

Comment:
Are these always going to be football related ? If not could `footballNavItem` be called `tertiaryNavItem` ?